relay.go

```diff
@@ -184,10 +184,12 @@ type Relayer struct {
 	// It allows timer re-use, while allowing timers to be created and started separately.
 	timeouts *relayTimerPool
 
-	peers   *RootPeerList
-	conn    *Connection
-	logger  Logger
-	pending atomic.Uint32
+	peers      *RootPeerList
+	conn       *Connection
+	relayConn  *relay.Conn
+	remoteAddr string
```

Comment:
Do you need a separate field `remoteAddr` since it's also in the `relayConn`?

Reply:
Unused field from a different variation of the change, removed. Thanks!